samples/config_helper_vulkan.cc

```diff
@@ -183,7 +187,7 @@ amber::Result NamesToVulkanFeatures(
     } else if (name == "inheritedQueries") {
       features->inheritedQueries = VK_TRUE;
     } else {
-      return amber::Result("Sample: Unknown Vulkan feature");
+      return amber::Result("Sample: Unknown Vulkan feature: " + name);
```

Comment:
this compiles!?
TIL

I expected the LHS of the + to be a std::string

Reply:
amber::Result takes a `const std::string&` as input. So, it's probably converting the first const char* to std::string then doing the add.